.github/workflows/ci-workflow.yml

```
@@ -0,0 +1,29 @@
+      - 'main'
+      - 'develop'
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest     # OS 지정
+    steps:
+      - uses: actions/checkout@v4
+      - name: ☕️ set up JDK 17        
+        uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)